src/helpers.ts

```diff
@@ -54,15 +54,24 @@ export function pathToConfig(pluginSettings: IPluginSettings) {
 }
 
 export function pathToJestPackageJSON(pluginSettings: IPluginSettings): string | null {
-  const defaultPath = normalize('node_modules/jest/package.json')
-  const craPath = normalize('node_modules/react-scripts/node_modules/jest/package.json')
+  let pathToNodeModules = join(pluginSettings.rootPath, 'node_modules')
 
+  if (pluginSettings.pathToJest) {
+    const relativeJestCmd = pluginSettings.pathToJest.split(' ')[0]
```

Comment:
Do we need to handle the case when the first arg is quoted? :sweat:

Reply:
yep 😩

Reply:
Well, hey ... the bright side is we can use the same test case to fix [`Process.js`](https://codecov.io/gh/facebook/jest/src/master/packages/jest-editor-support/src/Process.js#L28) in `jest-editor-support`. :fire: